### PR TITLE
[Keyboard] Add alas keyboard

### PR DESCRIPTION
**Pull request by yiancar**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by yiancar** [comment]
errors of CI have to do with community layouts

File: keyboards/alas/config.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
```
Comment (waffle87) [inline keyboards/alas/config.h:21]: Moving away from this as per #19714.
```suggestion
```

File: keyboards/alas/config.h [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * NKRO may prevent your keystrokes from being detected in the BIOS, but it is
+ * fully operational during normal computer usage.
+ *
+ * For a less heavy-handed approach, enable NKRO via magic key (LShift+RShift+N)
+ * or via bootmagic (hold SPACE+N while plugging in the keyboard). Once set by
+ * bootmagic, NKRO mode will always be enabled until it is toggled again during a
+ * power-up.
+ *
+ */
+//#define FORCE_NKRO
```
Comment (waffle87) [inline keyboards/alas/config.h:65]: Moving away from this as per #19718.
```suggestion
```

File: keyboards/alas/readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Alas
+
+This is a standard TKL layout PCB. It supports VIA.
```
Comment (waffle87) [inline keyboards/alas/readme.md:3]: > TKL

I'd fix this to avoid any confusion..

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/alas/config.h (hunk 1 above)
Comment (yiancar) [reply keyboards/alas/config.h:21]: good to know!

File: keyboards/alas/readme.md (hunk 3 above)
Comment (yiancar) [reply keyboards/alas/readme.md:3]: i thought i fixed this upsie one second

**Comment by yiancar** [comment]
ping:)

File: keyboards/alas/config.h [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+#define MATRIX_ROW_PINS { A2, B9, B8, B5, B4 }
+#define MATRIX_COL_PINS { A1, B1, B0, A7, A6, A5, A4, A3, B2, B10, B11, B12, B13, B14 }
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
```
Comment (fauxpark) [inline keyboards/alas/config.h:28]:
```suggestion
```

File: keyboards/alas/config.h [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+#define LED_PIN_ON_STATE 0
+
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* If defined, GRAVE_ESC will always act as ESC when CTRL is held.
+ * This is userful for the Windows task manager shortcut (ctrl+shift+esc).
+ */
+// #define GRAVE_ESC_CTRL_OVERRIDE
+
```
Comment (fauxpark) [inline keyboards/alas/config.h:43]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/alas/readme.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Alas
+
+This is a standard 60% layout PCB. It supports VIA.
+
+* Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
+* Hardware Supported: A 60% keyboard with STM32F072CB
+* Hardware Availability: https://geekhack.org/index.php?topic=119025.0 
+
+![Layer 0](https://i.imgur.com/LL5mZXA.png)
+![Layer 1](https://i.imgur.com/RJDVmty.png)
```
Comment (drashna) [inline keyboards/alas/readme.md:10]:
```suggestion
![Layer 0](https://i.imgur.com/LL5mZXAh.png)
![Layer 1](https://i.imgur.com/RJDVmtyh.png)
```

**Review by drashna — APPROVED** [review]

**Pull request merged**